versa/utterance_metrics/pseudo_mos.py

```diff
@@ -92,10 +92,17 @@ def pseudo_mos_setup(
         elif predictor == "singmos":
             torch.hub.set_dir(cache_dir)
             singmos = torch.hub.load(
-                "South-Twilight/SingMOS:v0.2.0", "singing_ssl_mos", trust_repo=True
+                "South-Twilight/SingMOS:v0.3.0", "singing_ssl_mos", trust_repo=True
```

Comment:
we can keep 0.2.0 for the old one?

Reply:
I keep the same setting in v0.3.0 as what we set in v0.2.0. So it's OK for update to v0.3.0.

Reply:
To update a new version of SingMOS, I additionally add a new model while keeping the old setting.

Reply:
Sounds good, in that case, please change back for this line. Thanks!